Python/ogusa/SS.py

```diff
@@ -195,7 +195,7 @@ def Euler_equation_solver(guesses, r, w, T_H, factor, j, tax_params, params, chi
 
 def SS_solver(b_guess_init, n_guess_init, wguess, rguess, T_Hguess,
               factorguess, chi_n, chi_b, tax_params, params, iterative_params, tau_bq,
-              rho, lambdas, weights, e, fsolve_flag):
+              rho, lambdas, weights, e, fsolve_flag=False):
```

Comment:
There are some calls to `SS_solver` with this flag and some without. The one call with this flag sets `fsolve_flag` to True, so I chose a default of False.